docs/user/faq.rst

```diff
@@ -852,7 +852,9 @@ cookie in subsequent requests.
 
 (See also the :ref:`cookie documentation <cookie-secure-attribute>`.)
 
-How can I serve a downloadable file with falcon?
+.. _serve-downloadable-as:
+
+How can I serve a downloadable file with Falcon?
```

Comment:
Maybe we could link the example from here also?

Reply:
Sure.